src/driver/amdxdna/npu1_message.c

```diff
@@ -416,8 +418,10 @@ int npu1_config_cu(struct amdxdna_hwctx *hwctx)
 	req.num_cus = hwctx->cus->num_cus;
 
 	ret = npu_send_msg_wait(xdna, chann, &msg);
-	if (ret == -ETIME)
-		npu1_destroy_context(xdna->dev_handle, hwctx);
+	if (ret == -ETIME) {
+		xdna_mailbox_stop_channel(chann);
+		xdna_mailbox_destroy_channel(chann);
```

Comment:
No need to do below here?
hwctx->priv->mbox_chann = NULL;
hwctx->fw_ctx_id = -1;

Reply:
we should do npu1_destroy_context(), since this channel doesn't response.